.github/workflows/_test_windows.yaml

```diff
@@ -0,0 +1,29 @@
+name: Unit Tests on Windows
+
+on:
+  workflow_call:
+    inputs:
+      original_repo_name:
+        required: true
+        type: string
+
+jobs:
+  test-windows:
+    name: Run unit tests on Windows
```

Comment:
I think that we could simplify this , removing the specific path and just calling go test .src/... from the action

Reply:
i've done a similar thing here, https://github.com/newrelic/coreint-automation/pull/44/files#diff-5ff564bce23726e3f2a53af8738d651130ca190b0c08f97bb6cbbbeeb79ac7b5R68